### ISSUE TITLE
Export NeonDbError

### ISSUE THREAD

**Issue by timclark97** [neondatabase/serverless#42]
Can [NeonDbError](https://github.com/neondatabase/serverless/blob/main/export/index.ts#L251) be included in the package exports to allow for [instanceof narrowing](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#instanceof-narrowing) in `catch` blocks like this.

```ts
import { NeonDbError } from '@neondatabase/serverless';
...
catch (e: unknown) {	
  if (e instanceof NeonDbError) {
     // Do something
  }
}
...
```

**Comment by jawj** [comment]
Absolutely! I thought I'd done this already, but it must have slipped through the net. I'll release a new driver version shortly.

**Comment by jawj** [comment]
This is now in `@neondatabase/serverless@0.5.7`. Do let me know if you have any trouble with it.

**Comment by timclark97** [comment]
Thank you!
